tasks/service-runner/app/service/service-runner.ts

```diff
@@ -285,28 +271,27 @@ export async function runServiceFromPull(workItem: WorkItemRecord, workItemLogge
               resolve({ batchCatalogs: catalogs });
             } else {
               clearTimeout(timeout);
-              const logErr = await _getErrorMessage(status, stdOut.aggregateLogStr, catalogDir, workItemLogger);
-              const errMsg = `${sanitizeImage(env.harmonyService)}: ${logErr}`;
+              const logErr = await _getErrorMessage(status, catalogDir, workItemLogger);
+              const errMsg = `${serviceName} - ${logErr}`;
```

Comment:
Changed this to a `-` in between the message and the service name, since another `:` gets added to the error message upstream. (The message ends up looking pretty awkward with two colons.)

Reply:
E.g.:
"WorkItem [3583761] failed with error: nasa/harmony-gdal-adapter:1.2.9: Incompatible variables"
will now look like:
"WorkItem [3583761] failed: nasa/harmony-gdal-adapter:1.2.9 - Incompatible variables"